frontend/src/admin/settings/settings.controllers.js

```diff
@@ -105,7 +105,15 @@ const SECTIONS = [
                 type: "string",
                 required: true,
                 autoFocus: true
-            }
+            },
+            {
+                key: "metabot-enabled",
+                display_name: "Metabot",
+                type: "boolean",
```

Comment:
I'd prefer making `type` be part of the `defsetting` macro (i.e., part of the settings themselves) so we don't need to add special cases for things on the frontend